vrs/vrs-annotation-batch-wrapper.py

```diff
@@ -21,9 +22,12 @@
 import hail as hl
 import hailtop.batch as hb
 from gnomad.resources.grch38.gnomad import public_release
-from gnomad_qc.v3.resources.annotations import vrs_annotations as v3_vrs_annotations
 from tgg.batch.batch_utils import init_job
```

Comment:
using isort 'from tgg.batch.batch_utils import init_job' should be the last import

Reply:
Noted! Will make sure the right order of imports is included in my next commit